### ISSUE TITLE
Police Oversight form: Thank you form landing page title should be H1

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#1505]
Fix the headings on the landing page on the thank you form so they are semantically correct and match the complaint form: 

- [x] Title is h2, should be h1
- [x] "Select an option" is h4, should be h2 
- [x] Headings in accordions, e.g. _Online_ --> "Required information" are h4s, should be h3s
- [x] "About us" is h3, should be h2

![Screen Shot 2019-03-15 at 5 07 41 PM](https://user-images.githubusercontent.com/1463708/54464891-9b484b80-4746-11e9-8747-7fb187550166.png)

<img width="790" alt="Screen Shot 2019-03-15 at 5 05 50 PM" src="https://user-images.githubusercontent.com/1463708/54464898-a3a08680-4746-11e9-82e9-f5db46d676f1.png">

**Comment by easherma** [comment]
Done

**Comment by annieszot** [comment]
Following @easherma's "Done" comment and closing issue
